app/models/base_organisation.rb

```diff
@@ -54,5 +57,32 @@ def setup_categories
   def has_relation?(category)
     self.category_organisations.any? {|v| v.category_id == category.id}
   end
+  
+  def slug_candidates
+    [
+      :short_name,
+      :prolonged_name,
+      :orged,
+      :name
+    ]
+  end
```

Comment:
I don't know @tansaku, but in the documentation of the gem they used this way - http://norman.github.io/friendly_id/file.Guide.html#Candidates.